CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.2.1
+
+- Added support for windows projects on [Flutter 3.7.7](https://github.com/flutter/flutter/releases/tag/3.7.7)
+- Added pubspec.lock to .gitigore, [see](https://dart.dev/guides/libraries/private-files#pubspeclock) 
+
```

Comment:
As mentioned before, we'll have to remove this.

Reply:
Resolved in [new commit](https://github.com/obemu/package_rename/commit/23fccf09eeb92780e0bd0e10b45c231503e02128)